TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+in conjuction with the Knative mark.
+
+The governance of Knative is an open, living document, and will continue to
+evolve as the community and project change. We expect over time we will adapt
+the way we run this committee, based on feedback from the community.
+
+- [Charter](#charter)
+- [Delegated Authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
+- [Committee Members](#committee-members)
+- [Decision Process](#decision-process)
+- [Getting in touch](#getting-in-touch)
+- [Composition](#composition)
+  - [Maintaining the Trademark Committee](#maintaining-the-trademark-committee)
+
+## Charter
+
+1. Advise the Trademark owner on the definition and enforcement of the "Knative"
+   trademark usage and branding guidelines.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+
+1. Review and approve the scope and definition of additional derivative Knative
+   marks (e.g. Knative Serving, Knative Core, etc). It is expected that the
+   Trademark Committee will delegate one or more marks to the steering committee
+   to provide branding for nascent efforts which are not yet subject to
+   conformance and third-party licensing.
+
+1. Only the Trademark Committee can redefine its own charter, membership, and
+   authority over Knative Trademark, conformance, and brand decisions.
+
+## Delegated authority
+
+The Trademark Committee may choose to create and delegate its authority to other
+committees as-needed. This may be used (for example) to create sub-committees
+for specific technical or marketing questions related to conformance or joint
+marketing.
+
+## Committee Meetings
+
+The committee meets as-needed. Meetings are held online.
+
+Given the private nature of some of these discussions (e.g. trade secrets, legal
+and compliance concerns, etc.) some meetings are held in private.
+
+Meeting notes are available to members of the
+[knative-dev mailing list](https://groups.google.com/forum/#!forum/knative-dev)
+(note: these are accessible to members of the knative-users list as well).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as
+[issues in the community repo](https://github.com/knative/community/issues), and
+the trademark committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee Members
+
+| Member                   | Organization | Profile |
+| ------------------------ | ------------ | ------- |
+| Unknown - to be selected | Google       | Unknown |
+| Unknown - to be selected | IBM/RedHat   | Unknown |
+| Unknown - to be selected | VMware       | Unknown |
+
+## Decision process
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding assignment of the mark, charter,
+conformance, and scope.
+
```

Comment:
Insert: 
Decisions requiring a vote include: issuing written policy, amending existing written policy, official responses to publicly raised issues, or any other decisions that are in the scope of the Trademark Committee’s charter.

Voted decisions pass when all members vote to approve the decision, or a majority approves and the rest of the members abstain.

Reply:
Done.